components/modules/utils/config/audioPlayer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect, useRef, FC } from "react";
+import AudioControls from "./audioControls";
+
+type TrackItems={
+  tracks: any;
+  [index: number]:{title: string;
+  artist: string;
+  audioSrc: string;
+  image: string;}
+}
+
+const AudioPlayer= ({tracks}:TrackItems) => {
```

Comment:
As this is a component we can keep the same notion:

```suggestion
const AudioPlayer: FC<TrackItems>= ({tracks}) => {
```